packages/react-charting/src/components/AreaChart/__snapshots__/AreaChart.test.tsx.snap

```diff
@@ -150,15 +150,15 @@ exports[`AreaChart - mouse events Should render callout correctly on mouseover 1
             cx={40}
             cy={115.625}
             data-is-focusable={true}
-            fill="#ffffff"
+            fill="red"
```

Comment:
Are these test cases flaky?

Reply:
Yes. There seems to be some flakiness how the x y values changed for this file.